Formula/i/ipopt.rb

```diff
@@ -21,26 +21,7 @@ class Ipopt < Formula
   depends_on "ampl-mp"
   depends_on "gcc" # for gfortran
   depends_on "openblas"
-
-  resource "mumps" do
-    # follow links provided in official repo: https://github.com/coin-or-tools/ThirdParty-Mumps/blob/stable/3.0/get.Mumps
-    url "http://coin-or-tools.github.io/ThirdParty-Mumps/MUMPS_5.6.2.tar.gz"
-    mirror "http://deb.debian.org/debian/pool/main/m/mumps/mumps_5.6.2.orig.tar.gz"
-    sha256 "13a2c1aff2bd1aa92fe84b7b35d88f43434019963ca09ef7e8c90821a8f1d59a"
-
-    patch do
-      # MUMPS does not provide a Makefile.inc customized for macOS.
-      on_macos do
-        url "https://raw.githubusercontent.com/Homebrew/formula-patches/ab96a8b8e510a8a022808a9be77174179ac79e85/ipopt/mumps-makefile-inc-generic-seq.patch"
-        sha256 "0c570ee41299073ec2232ad089d8ee10a2010e6dfc9edc28f66912dae6999d75"
-      end
-
-      on_linux do
-        url "https://gist.githubusercontent.com/dawidd6/09f831daf608eb6e07cc80286b483030/raw/b5ab689dea5772e9b6a8b6d88676e8d76224c0cc/mumps-homebrew-linux.patch"
-        sha256 "13125be766a22aec395166bf015973f5e4d82cd3329c87895646f0aefda9e78e"
-      end
-    end
-  end
+  depends_on "mumps"
```

Comment:
`depends_on "mumps"` should be put before `depends_on "openblas"` as  mumps implies openblas.

Reply:
Amended and pushed.

Reply:
Great. I am afraid that now compilation of ipopt fails with:
``` Error: inreplace failed
  Makefile.inc:
    expected replacement of "@rpath/" with "/opt/homebrew/opt/mumps/lib/"```
   Is the formula working on your local installation?

Reply:
Admittedly I only tested mumps as that's what I was focused on, I'll verify ipopt now.

Reply:
On further inspection, looks like it's actually `mumps` failing, not `ipopt`. I've been testing on Linux, looks like the failure is coming from a MacOS logical branch. I've pushed an update.